src/cdk/overlay/dispatchers/overlay-keyboard-dispatcher.spec.ts

```diff
@@ -178,6 +183,21 @@ describe('OverlayKeyboardDispatcher', () => {
     expect(overlayTwoSpy).not.toHaveBeenCalled();
     expect(overlayOneSpy).toHaveBeenCalled();
   });
+
+  it('should not run change detection if there are no `keydownEvents` observers', () => {
+    spyOn(appRef, 'tick');
+    const overlayRef = overlay.create();
+    keyboardDispatcher.add(overlayRef);
+
```

Comment:
Should we have a similar test on the click dispatcher?

Reply:
Yep, done.